app/controllers/concerns/fully_authenticatable.rb

```diff
@@ -8,7 +8,7 @@ def confirm_two_factor_authenticated(id = nil)
   end
 
   def delete_branded_experience
-    ServiceProviderRequest.from_uuid(sp_session[:request_id]).delete
+    ServiceProviderRequestProxy.delete(sp_session[:request_id])
```

Comment:
The delete here can actually be removed to increase client/performance.  However we need to make sure we have enough space on redis to handle the data that accumulates under the TTL